astroquery/xmatch/tests/test_xmatch_remote.py

```diff
@@ -20,7 +20,7 @@
 from ...xmatch import XMatch
 
 
-DATA_DIR = os.path.join(os.path.dirname(__file__), 'data')
+DATA_DIR = Path(__file__).parent / "data"
```

Comment:
would this work on windows? Could we maybe keep the `os.path.join` approach here and everywhere below to handle separators?

Reply:
yes, in VM-ed windows. Would it also be true in real user cases? I would really rather avoid getting rid of the proper way of handling separators just because it works for us in the one case we set up in CI (historically it indeed was a problem). 
I'm not against using path-like objects, but would rather see they are stuck into `os.path.join` (it can receive Path objects) rather than having a hard-wired `/` as a separator.

pinging @pllim as the windows expert to comment whether my concern is outdated.

Reply:
`/` operation for `Path` should be OS agnostic. So 👍 from me. 😸

Reply:
OK, let's have this in then. fyi I still don't like this approach, and would prefer to stick with os.path (most of all as that's what we use elsewhere (along with the odd manual string handling, all of which should be cleaned up eventually), but the PR fixes part of the problems we had, so will live with it.